static/templates/left_sidebar.hbs

```diff
@@ -69,6 +55,39 @@
                 <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
         </ul>
+
+        <div id="private_messages_section" class="zoom-out hidden-for-spectators">
+            <a id="hide-more-private-conversations">
+                <i class="fa fa-chevron-left" aria-hidden="true"></i>
+                {{t 'BACK' }}
+            </a>
+            <div id="private_messages_list">
+                <div id="default_private_messages_view_header" class="private-messages-section-header zoom-in-hide">
+                    <div id="toggle-private-messages-section">
+                        <i id="toggle-private-messages-section-icon" class="fa fa-caret-down fa-lg toggle_sidebar_section_icon" aria-hidden="true"></i>
+                        <h4 class="sidebar-title" data-tippy-content="{{t 'Private messages' }}">{{t 'PRIVATE MESSAGES' }}</h4>
+                    </div>
+                    {{~!-- squash whitespace --~}}
+                    <span class="unread_count"></span>
+                    <a href="#narrow/is/private">
+                        <i id="show-all-private-conversations-icon" class="fa fa-align-right tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'All private messages (P)'}}"></i>
```

Comment:
When the PM section is collapsed, this gets hidden. I'm not totally sure what the solution is, but maybe forcing the tooltip to go down would work?

Reply:
Hmm, I too came across this while playing more with the PR -- Let me see to fix this up.

Reply:
Looks like this is still an issue - can you avoid resolving conversations until you've addressed the issue described in the comment? 

And let me know if you want any help brainstorming solutions to this :)

Reply:
OK, I've fixed this thing --
Yeah -- I too thought that this should be fixed by forcing the tool-tip to place at `bottom`, now for doing that the first and easiest way came in my mind was using - `data-tippy-placement="bottom"` but unfortunately, even after forcing the tooltip to go bottom -- it was not visible completely, so last option remaining was to put it in `tippy.js` and handle the placement also from there itself.

Reply:
Thanks @evykassirer for reporting this!

Reply:
looks great, thanks!